Reference/conformance/conformance.pb.swift

```diff
@@ -230,11 +230,28 @@ struct Conformance_ConformanceResponse: SwiftProtobuf.Message {
   var unknownFields = SwiftProtobuf.UnknownStorage()
 
   enum OneOf_Result: Equatable {
```

Comment:
Oh, sorry, I thought you meant put comments on the *cases*.

Having a comment on the enum might be valuable. Having it on the property is less valuable since the type is easily findable from the property definition, but the property isn't always easily findable from the enum definition.